src/sp/it/pl/util/system/Environment.kt

```diff
@@ -327,18 +317,18 @@ private fun Desktop.Action.isSupportedOrWarn() =
             false
         }
 
-private fun File.openWindowsExplorerAndSelect() {
-    try {
-        Runtime.getRuntime().exec(arrayOf("explorer.exe", "/select,", "\"$path\""))
-    } catch (e: IOException) {
-        logger.error(e) { "Failed to open explorer.exe and select file=$this" }
-    }
-}
+private fun File.openWindowsExplorerAndSelect() =
```

Comment:
Why was return value added? It is unused and unneeded.

Reply:
Because if it fails, it just swallows the exception and continues as if nothing happens. Such a function should provide some way of feedback, even if we don't need it right now.